### DISCUSSION TITLE
Welcome to flood Discussions!

### DISCUSSION THREAD

**Discussion by sslivkoff** [paradigmxyz/flood#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DenhamPreen** [comment]
Hi flood 🌊 community, 

Denham here, working on [Envio](https://envio.dev), here to leverage this awesome technology to benchmark RPC latency specifically 👋 .  

Hacking on an idea 💡 at the moment that will hopefully speed up RPC significantly and using flood to benchmark these findings. 
 - Problem: RPC is slow due to sending data in large payloads
 - Solution: Run a local HTTP proxy server that translates the RPC API query into our HyperSync API query, eliminating large payloads allowing plug and play RPC replacement with significant performance improvements. 
 
Introducing myself here first before opening a discussion around whether the latency metrics recorded should include the unsuccessful requests or whether this misleadingly skews the findings.
